sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/ProjectScanContainer.java

```diff
@@ -234,13 +234,14 @@ protected void doAfterStart() {
     }
     String branch = tree.root().definition().getBranch();
     if (branch != null) {
-      LOG.info("Branch key (deprecated): {}", branch);
+      LOG.info("Branch key: {}", branch);
+      LOG.warn("The use of \"sonar.branch\" is deprecated and replaced by \"sonar.branch.name\". See https://redirect.sonarsource.com/doc/branches.html.");
```

Comment:
https://redirect.sonarsource.com/doc/branches.html doesn't exist, is it expected ?